Engine/source/gfx/bitmap/ddsLoader.cpp

```diff
@@ -607,7 +607,7 @@ bool DDSFile::read(Stream &s, U32 dropMipCount)
       else if ( mFlags.test( PitchSizeFlag ) )
          mPitchOrLinearSize = getSurfacePitch( dropMipCount );
       else
-         mPitchOrLinearSize = mPitchOrLinearSize; // Do nothing?
+         ; // Do nothing?
```

Comment:
I left it just as an anchor for the comment, but I guess there's not much point - I can get rid of the else entirely (the var is just a u32). I guess the point of the self-assign was to be able to set a breakpoint there, but if someone really needs that they can always add it back locally.